cmd/tcpgoon.go

```diff
@@ -15,6 +16,7 @@ import (
 
 type tcpgoonParams struct {
 	hostPtr              string
```

Comment:
do the "Ptr" names make sense any more? If they are just strings/ints, and not pointers to strings/ints (as happened with the args mgmt methods in the standard library), I'd rename